.toolkitrc.yml

```diff
@@ -0,0 +1,23 @@
+plugins:
+  - "@dotcom-tool-kit/component"
+  - "@dotcom-tool-kit/webpack"
+  - "@dotcom-tool-kit/node"
+  - "@dotcom-tool-kit/jest"
+hooks:
+  test:local:
+    - JestLocal
+  build:local:
+    - WebpackDevelopment
+  run:local:
+    - Node
+options:
+  "@dotcom-tool-kit/circleci":
+    nodeVersion: 16.14.0
+  "@dotcom-tool-kit/vault":
+    team: "next"
+    app: "next-newsletter-signup"
```

Comment:
[Observation] Vault has both `next-newsletter-signup` and `n-newsletter-signup`. Are we sure it's the `next-` that we need here when the repo in `n-`?

Reply:
There is nothing in `n-newsletter-signup`. It is basically empty. 
```
//teams/next/n-newsletter-signup/continuous-integration
{
  "circleci": {
    "preset": "component"
  },
  "env": {}
}
```

```
//teams/next/n-newsletter-signup/meta
{
  "keys": {},
  "schemaVersion": 1,
  "team": "accounts-customer-products"
}
```

Reply:
So from that, I assumed it must be "next-newsletter-signup"

Reply:
`next-newsletter-signup` (https://github.com/Financial-Times/next-newsletter-signup) looks to be "A sign-up page for FT newsletters that are in front of the paywall." so sounds like a completely different thing. I think Andy Gout is working on it, may be worth a quick word with him.